### DISCUSSION TITLE
automatic modification of footnote numbers due to the visual editor  of VS Studio Code

### DISCUSSION THREAD

**Discussion by mrossde** [quarto-dev/quarto-cli#8773] [Q&A]
### Description

Hello, 

My Setup: 
Quarto 1.4.549
VS Studio Code 1.86.1
Quarto Extension v1.110.1

i have worked on several documents for a lecture and want the revealjs output. When i include the documents in one document, the footnotes break, probably because of the automatic numbering, which the visual editor sets (e.g. Text [^38]  and the footnote [^38]: vgl. @North.2011, S. 16

When I use Footnotes with the syntax `^[footnote see @source.2023]`. , the editor directly transforms this into ongoing footnotes. 
**Is it possible to disable the function in the visual editor?**

Thanks in advance for answering my question :-)
Best Regards, Mario

**Comment by mcanouil** [comment]
Could you share a small **self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

Maybe a screen recording also to show exactly how to reproduce.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

![A placeholder image](https://placehold.co/600x400.png)

The end.
````
`````

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.
